sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_quickpulse/_exporter.py

```diff
@@ -73,8 +103,36 @@ def export(
         :return: The result of the export.
         :rtype: ~opentelemetry.sdk.metrics.export.MetricExportResult
         """
-        # TODO
-        return MetricExportResult.SUCCESS
+        result = MetricExportResult.SUCCESS
+        base_monitoring_data_point = kwargs.get("base_monitoring_data_point")
+        if metrics_data is None or base_monitoring_data_point is None:
```

Comment:
What does it mean if the data point is none? Just that there are no metrics to report?

Reply:
`metrics_data` shouldn't be `None`. Removed check.